src/targets/gpu/time_op.cpp

```diff
@@ -45,23 +45,40 @@ std::vector<argument> generate_arguments(const std::vector<shape>& shapes,
 }
 
 template <class F>
-double time_loop(migraphx::gpu::context& gctx, int n, F f)
+double time_loop(migraphx::gpu::context& gctx, int m_iter, int n_loop, F f)
```

Comment:
I think the variable `m_iter` is really confusing since this could refer to the number of iterations of the outer loop or inner loop. I still think `batch` is a much better name since you are launching the kernels in a batch. However, if you dont like that name, I guess maybe `group` or `bundle` could be used. 

The `n_loop` should be called something more like `nruns` or `runs`.

Reply:
Ok, I'm thinking of changing `m_iter` to `bundle` and `n_loop` to `runs`.